src/roiextractors/segmentationextractor.py

```diff
@@ -270,13 +272,31 @@ def get_num_channels(self):
     def get_num_planes(self):
         """
         Returns the default number of planes of imaging for the segmentation extractor.
-        Detaults to 1 for all but the MultiSegmentationExtractor
+        Defaults to 1 for all but the MultiSegmentationExtractor
         Returns
         -------
         self._num_planes: int
         """
         return self._num_planes
 
+    def frame_to_time(self, frame_indices: Union[IntType, ArrayType]) -> Union[FloatType, ArrayType]:
```

Comment:
ArrayType includes int I think.

What do you think of having `frame_indices=None` means getting all the times?

Reply:
> ArrayType includes int I think.

If you examine the full nesting of `ArrayLike` it does not contain our exact definition of `IntType`, though comes close to the spirit.

However, annotation typing is really just for us to be able to look at this signature and understand what it accepts. As I read it, I can specify numpy slicing semantics (single integer or array or integers) to get those frames as times.

Adding `None` to that would also reinforce numpy-like semantics, yes.

As for whether or not we want to support numpy semantics in general (as per previous conversations about the imaging extractor `get_frames`, that's the more central question I think.

Reply:
It is not about numpy-like semantics which I think would not apply here.

It is more so it is consistent with other methods of `segmentation_extractor` where passing an argument as None means give me at all of it. See `get_roi_pixel_masks`.

Reply:
Concerning the typing issue that's fine if you think is more clear to read. However, while I have not checked the nesting the documentation seems to be clear that it should contain scalars such as a int:

https://numpy.org/devdocs/reference/typing.html#numpy.typing.ArrayLike